frontend/search/searchController.js

```diff
@@ -25,10 +26,19 @@
             promise.then(function success(response) {
                 searchCtrl.institutions = response;
                 searchCtrl.loading = true;
+                searchCtrl.hasChanges = true;
             });
             return promise;
         };
 
+        searchCtrl.setHasChanges = () => {
+            if(searchCtrl.search_keyword)
```

Comment:
do you agree that this is the same of: `searchCtrl.hasChanges = !!searchCtrl.search_keyword` or `searchCtrl.hasChanges = Boolean(searchCtrl.search_keyword)` ?